scripts/predict.py

```
@@ -0,0 +1,177 @@
+            if not self.scaler_path.exists():
+                raise FileNotFoundError(f"Scaler file not found at {self.scaler_path}")
+            if not self.features_path.exists():
+                raise FileNotFoundError(f"Features file not found at {self.features_path}")
+            
+            self.model = joblib.load(self.model_path)
+            self.scaler = joblib.load(self.scaler_path)
+            
+            features_df = pd.read_csv(self.features_path, header=None)
+            self.features = features_df.iloc[:, 0].tolist()
+            
+            print(f"Model artifacts loaded successfully")
```

Ruff (0.15.0)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)